src/ServiceProvider.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Justbetter\StatamicPageBuilderKit;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\File;
+use Statamic\Facades\Blueprint;
+use Statamic\Facades\Collection as CollectionFacade;
+use Statamic\Facades\Fieldset as FieldsetFacade;
+use Statamic\Facades\Site;
+use Statamic\Facades\YAML;
+use Statamic\Fields\Fieldset;
+use Statamic\Providers\AddonServiceProvider;
+use Statamic\Support\Str;
+
+class ServiceProvider extends AddonServiceProvider
+{
+    public function bootAddon(): void
+    {
+        $this
+            ->bootConfig()
+            ->bootViews()
+            ->bootPageBuilder();
```

Comment:
Maybe we can move all the code to a trait, to clean up the service provider a bit.